src/components/ButtonBox/index.jsx

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+import Button from "../Button/index.jsx";
+import styles from "./styles/styles.module.scss";
+
+const ButtonBox = ({ title }) => {
+  let buttonBox = (
+    <div className={styles.oneButtonBox}>
+      <Button title="Next" path="/programminglanguages"></Button>
+    </div>
+  );
+
+  if (title === "Programming languages") {
```

Comment:
Page title should not be the code's logic. Imagine that you want to fix a typo on page title. Then you have to refactor this part as well. In most cases, what we show to customers is not the data that we write the application's logic based on. 👈🏽 Consider this as a general rule.

Reply:
thanksssss , I I'm just starting to understand this.